gc/base/segregated/OverflowSegregated.cpp

```diff
@@ -154,7 +153,7 @@ MM_OverflowSegregated::handleOverflow(MM_EnvironmentBase *env)
 					/* scan overflowed object */
 					/* TODO Fix this as it is wrong for metronome GC policy */
 					/* The subclassing of markingscheme by segregatedmarking scheme needs a lot of work and means metronome is pretty broken */
-					_extensions->collectorLanguageInterface->markingScheme_scanObject(env, object, MM_CollectorLanguageInterface::SCAN_REASON_OVERFLOWED_OBJECT);
+					markingScheme->scanObject(env, object, MM_MarkingDelegate::SCAN_REASON_OVERFLOWED_OBJECT);
```

Comment:
I agree. But MarkingDelegate.hpp can't include MarkingScheme.hpp. I will move them to omrgcconsts.h.